packages/core/stories/chip.stories.js

```diff
@@ -0,0 +1,156 @@
+                  id="Shape"
+                />
+              </g>
+            </g>
+          </svg>
+          <span className="ray-chip__text">Furnished</span>
+        </div>
+      </div>
+      <div className="ray-chip ray-chip--active">
+        <div className="ray-chip__icon-wrapper">
+          <svg
+            className="ray-chip__icon"
+            width="14px"
+            height="16px"
+            viewBox="0 0 14 16"
+          >
+            <g id="building-group1" stroke="none" strokeWidth="1" fill="none">
+              <g
+                id="building-group2"
+                transform="translate(-289.000000, -380.000000)"
+                fill="#0000FF"
+              >
+                <g
+                  id="building-group3"
+                  transform="translate(272.000000, 368.000000)"
+                >
+                  <g
+                    id="building-group4"
+                    transform="translate(16.000000, 12.000000)"
+                  >
+                    <path
+                      d="M14.4,15.36 L13.44,15.36 L13.44,0.96 C13.4394356,0.430040593 13.0099594,0.000564387495 12.48,0 L3.52,0 C2.99004059,0.000564387495 2.56056439,0.430040593 2.56,0.96 L2.56,15.36 L1.6,15.36 C1.42326888,15.36 1.28,15.5032689 1.28,15.68 C1.28,15.8567311 1.42326888,16 1.6,16 L14.4,16 C14.5767311,16 14.72,15.8567311 14.72,15.68 C14.72,15.5032689 14.5767311,15.36 14.4,15.36 Z M9.6,15.36 L6.4,15.36 L6.4,10.88 L9.6,10.88 L9.6,15.36 Z M12.8,15.36 L10.24,15.36 L10.24,10.88 C10.24,10.5265378 9.95346224,10.24 9.6,10.24 L6.4,10.24 C6.04653776,10.24 5.76,10.5265378 5.76,10.88 L5.76,15.36 L3.2,15.36 L3.2,0.96 C3.20035211,0.783414936 3.34341494,0.640352115 3.52,0.64 L12.48,0.64 C12.6565851,0.640352115 12.7996479,0.783414936 12.8,0.96 L12.8,15.36 Z M10.88,5.76 L5.12,5.76 C4.76653776,5.76 4.48,6.04653776 4.48,6.4 L4.48,7.68 C4.48,8.03346224 4.76653776,8.32 5.12,8.32 L10.88,8.32 C11.2334622,8.32 11.52,8.03346224 11.52,7.68 L11.52,6.4 C11.52,6.04653776 11.2334622,5.76 10.88,5.76 Z M10.88,7.68 L5.12,7.68 L5.12,6.4 L10.88,6.4 L10.88,7.68 Z M10.88,1.92 L5.12,1.92 C4.76653776,1.92 4.48,2.20653776 4.48,2.56 L4.48,3.84 C4.48,4.19346224 4.76653776,4.48 5.12,4.48 L10.88,4.48 C11.2334622,4.48 11.52,4.19346224 11.52,3.84 L11.52,2.56 C11.52,2.20653776 11.2334622,1.92 10.88,1.92 Z M10.88,3.84 L5.12,3.84 L5.12,2.56 L10.88,2.56 L10.88,3.84 Z"
```

Comment:
is there a less crazy svg we could use so the test file isnt so cluttered lol

Reply:
yeah i can use a circle or something

Reply:
should be a small square now